next/README.md

```diff
@@ -1,73 +1,54 @@
-# 3D Tiles Next
-
-**3D Tiles Next** is a set of capabilities for the future of 3D Tiles. Currently, they take the form of extensions to the [3D Tiles 1.0](../specification) specification. In the future, these extensions may be incorporated into 3D Tiles 2.0.
-
-The 3D Tiles Next extensions add capabilities to 3D Tiles on a number of fronts:
-
-3D Tiles Next **enables new methods for storing metadata.** Metadata can now be stored at different levels of granularity: tileset, tile, group, feature, and texture. Texture metadata allows for fine-grained storage of metadata down to individual texels. Feature metadata is similar to the existing Batch Table format, but allows for additional types such as enumerated types (enums). Tileset, tile and group metadata provide standardized methods to store coarse-grained metadata about a tileset.
-
-<img src="figures/metadata-granularity.png" alt="Metadata Granularity" width="600">
-
-**Implicit Tiling** is an alternate method for describing a tileset that **enables new algorithms and optimizations**. This compact tree representation allows for new tree traversal algorithms, raycasting, and faster spatial queries. It also **provides interoperability** with several geospatial formats such as CDB, TMS, WMTS, and S2.
-
-<img src="figures/implicit-tiling.jpg" width="600px" alt="Implicit Tiling">
+<p align="center"><img src="./figures/next_logo.png" alt="3D Tiles Next" width="500px" align="center"></p>
```

Comment:
Just a heads-up, this logo might be tweaked after some feedback with Patrick.

Reply:
Sounds good. If there's a canonical version elsewhere in the repository I can just link to that, or maybe this is the right place for it?